backend/src/main/java/gov/cdc/usds/simplereport/db/repository/SpecimenBodySiteRepository.java

```diff
@@ -0,0 +1,14 @@
+package gov.cdc.usds.simplereport.db.repository;
+
+import gov.cdc.usds.simplereport.db.model.SpecimenBodySite;
+import java.util.List;
+
+/** Specification of EternalAuditedEntityRepository for {@link SpecimenBodySite} manipulation. */
+public interface SpecimenBodySiteRepository extends EternalAuditedEntityRepository<SpecimenBodySite> {
```

Comment:
need to update this from `EternalAuditedEntity` -> `AuditedEntity`

Reply:
I updated the model but forgot to update the repo.  This is done now.